pkg/security/ebpf/c/dentry_resolver.h

```diff
@@ -17,7 +17,7 @@
 
 #define DR_MAX_TAIL_CALL          30
 #define DR_MAX_ITERATION_DEPTH    45
-#define DR_MAX_SEGMENT_LENGTH     255
+#define DR_MAX_SEGMENT_LENGTH     245
```

Comment:
it's may due to a change in clang 14 but would need more time to investigate
The issue was found only on tracepoint_dentry_resolver_kern probe

Reply:
I have this https://github.com/DataDog/datadog-agent/pull/12728 to investigate on the CWS side